packages/perf/ios/RNFBPerf/RNFBPerfModule.m

```diff
@@ -73,7 +73,6 @@ + (BOOL)requiresMainQueueSetup {
                   : (RCTPromiseResolveBlock)resolve rejecter
                   : (RCTPromiseRejectBlock)reject) {
   [FIRPerformance sharedInstance].dataCollectionEnabled = (BOOL)enabled;
-  [FIRPerformance sharedInstance].instrumentationEnabled = (BOOL)enabled;
```

Comment:
Removing this here seems correct, but I wonder if we need to also export instrumentationEnabled in constantsToExport  so we can use it in the javascript layer